lib/mayaUsd/sceneIndex/proxyShapeSceneIndexPlugin.cpp

```diff
@@ -153,21 +170,30 @@ HdSceneIndexBaseRefPtr MayaUsdProxyShapeMayaNodeSceneIndexPlugin::_AppendSceneIn
             // Flatten transforms, visibility, purpose, model, and material
             // bindings over hierarchies.
             // Do it the same way as USDView does with a SceneIndices chain
-            HdSceneIndexBaseRefPtr _sceneIndex
+            HdSceneIndexBaseRefPtr sceneIndex
                 = UsdImagingPiPrototypePropagatingSceneIndex::New(usdImagingStageSceneIndex);
-            _sceneIndex = UsdImagingNiPrototypePropagatingSceneIndex::New(_sceneIndex);
+#if PXR_VERSION < 2311
```

Comment:
@seando-adsk : there is a missing case here which is for usd 23.08, it should contain : 
```
            //#For USD 23.08 and later, HD_API_VERSION=54 in USD 23.08
            static const bool _displayUnloadedPrimsWithBounds = true;

            HdContainerDataSourceHandle const stageInputArgs =
            HdRetainedContainerDataSource::New(
                UsdImagingStageSceneIndexTokens->includeUnloadedPrims,
                HdRetainedTypedSampledDataSource<bool>::New(
                    _displayUnloadedPrimsWithBounds));

            // Create the scene index graph.
            _sceneIndex = _stageSceneIndex =
                UsdImagingStageSceneIndex::New(stageInputArgs);

            static HdContainerDataSourceHandle const materialPruningInputArgs =
                HdRetainedContainerDataSource::New(
                    HdsiPrimTypePruningSceneIndexTokens->primTypes,
                    HdRetainedTypedSampledDataSource<TfTokenVector>::New(
                        { HdPrimTypeTokens->material }),
                    HdsiPrimTypePruningSceneIndexTokens->bindingToken,
                    HdRetainedTypedSampledDataSource<TfToken>::New(
                        HdMaterialBindingsSchema::GetSchemaToken()));

            // Prune scene materials prior to flattening inherited
            // materials bindings and resolving material bindings
            _sceneIndex = _materialPruningSceneIndex =
                HdsiPrimTypePruningSceneIndex::New(
                    _sceneIndex, materialPruningInputArgs);

            static HdContainerDataSourceHandle const lightPruningInputArgs =
                HdRetainedContainerDataSource::New(
                    HdsiPrimTypePruningSceneIndexTokens->primTypes,
                    HdRetainedTypedSampledDataSource<TfTokenVector>::New(
                        HdLightPrimTypeTokens()),
                    HdsiPrimTypePruningSceneIndexTokens->doNotPruneNonPrimPaths,
                    HdRetainedTypedSampledDataSource<bool>::New(
                        false));

            _sceneIndex = _lightPruningSceneIndex =
                HdsiPrimTypePruningSceneIndex::New(
                    _sceneIndex, lightPruningInputArgs);

            // Use extentsHint for default_/geometry purpose
            HdContainerDataSourceHandle const extentInputArgs =
                HdRetainedContainerDataSource::New(
                    UsdGeomTokens->purpose,
                    HdRetainedTypedSampledDataSource<TfToken>::New(
                        UsdGeomTokens->default_));

            _sceneIndex =
                UsdImagingExtentResolvingSceneIndex::New(
                    _sceneIndex, extentInputArgs);

            if (_displayUnloadedPrimsWithBounds) {
                _sceneIndex =
                    UsdImagingUnloadedDrawModeSceneIndex::New(_sceneIndex);
            }

            _sceneIndex = _rootOverridesSceneIndex =
                UsdImagingRootOverridesSceneIndex::New(_sceneIndex);

            _sceneIndex =
                UsdImagingPiPrototypePropagatingSceneIndex::New(_sceneIndex);
        
            _sceneIndex =
                UsdImagingNiPrototypePropagatingSceneIndex::New(_sceneIndex);

            _sceneIndex = _selectionSceneIndex =
                UsdImagingSelectionSceneIndex::New(_sceneIndex);

            _sceneIndex =
                UsdImagingRenderSettingsFlatteningSceneIndex::New(_sceneIndex);

            _sceneIndex =
                HdFlatteningSceneIndex::New(
                    _sceneIndex, UsdImagingFlattenedDataSourceProviders());

            _sceneIndex =
                UsdImagingDrawModeSceneIndex::New(_sceneIndex,
                                                  /* inputArgs = */ nullptr);

            _sceneIndex = _displayStyleSceneIndex =
                HdsiLegacyDisplayStyleOverrideSceneIndex::New(_sceneIndex);
```

Reply:
@dj-mcg can you have a look at the code review comment here.

Reply:
@dj-mcg Dan would you mind if we pushed the change to the missing 23.08 case to this PR? We are using 23.08 internally and need this fix for MayaHydra.

Reply:
Oh, so sorry to have dropped the ball on this, I was OOO and am just catching up on things.

No problem checking this in, thanks for taking care of this case!